README.md

```diff
@@ -114,6 +114,7 @@ installations requiring long-term consistency.
 | [no-commented-out-tests][]   | Disallow commented out tests                                      |                  |                     |
 | [no-duplicate-hooks][]       | Disallow duplicate hooks within a `describe` block                |                  |                     |
 | [no-empty-title][]           | Disallow empty titles                                             |                  |                     |
+| [no-export][]                | Disallow export from test files                                   | ![recommended][] |                     |
```

Comment:
Not recommended (changing recommended rules is a breaking change)

Reply:
what's the process of getting it to be a recommended change?  This rule prevents very hard to track down problems....

Reply:
Just remembering when we release a new major :)

Reply:
i'll remove it for now, but I would like to move this to recommended in another PR, if you don't mind providing guidance on that process.

Reply:
Sure! Just open up a PR updating the readme and add it to the config exported from `index` and I'll make sure to merge it before making a major release